static/src/javascripts/projects/common/views/svg.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+export default (
+    markup: string,
+    classes: ?Array<string>,
+    title: ?string
```

Comment:
`title` is not just nullable but optional, right? So you could use `title?: string`?